### PR TITLE
Added new --no-config option to GetBinaryCommand, to skip generating config

### LINKED ISSUE #37: Allow GetBinaryCommand to dowanload biunary without generating a config file sample
> Originally reported in https://github.com/roadrunner-server/roadrunner/discussions/1287

I have a project where roadrunner's config already exists in a different folder than the one where the rr binary is downloaded, and I pass it t with `-c path/to/.rr.yml` when starting the server.

Would you be open to add a flag (let's say `--no-config`, or something in those lines) to the `./vendor/bin/rr get` command to not generate a config file?

If so, I'm open to contribute this.

### PR DESCRIPTION
**Pull request by acelaya**
Closes #37 

This adds a new `--no-config` option to the GetBinaryCommand so that config generation can be skipped even in no-interactive mode.

Since it's a simple flag I have defined it inline, inside GetBinaryCommand, but I can switch it to a separated Option class as with `os`, `arch`, `version`, etc.

### CODE REVIEW

**Review by butschster — APPROVED** [review]

**Review by msmakouz — APPROVED** [review]

**Comment by acelaya** [comment]
Hey!

Considering there has been two reviews and two approvals, is there any reason to not merge this?

**Comment by acelaya** [comment]
@butschster @msmakouz could we get this landed?

**Review by butschster — APPROVED** [review]

**Pull request merged**
